public/video-ui/src/components/Header.js

```diff
@@ -18,15 +18,39 @@ export default class Header extends React.Component {
     }
   }
 
-  renderHomeAndSearch() {
+  renderHome() {
     return (
       <div className="flex-container topbar__global">
         <Link to="/" className="topbar__home-link" title="Home"></Link>
+      </div>
+    );
+  }
+
+  renderSearch() {
+    return (
+      <div className="flex-container topbar__global">
         <VideoSearch {...this.props}/>
       </div>
     );
   }
 
+  renderHeaderBack() {
```

Comment:
would `renderBack` be explicit enough as we're in the `Header` component already?